docs/01-overview/02-main-areas/application-connectivity/ac-06-api-registration.md

```diff
@@ -0,0 +1,23 @@
+---
+title: API registration in Application Registry
```

Comment:
Isn't it more like an operation or technical reference?

Reply:
Moved to Operations.